v0/destinations/freshmarketer/data/FRESHMARKETERIdentifyConfig.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "destKey": "emails",
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "first_name",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "last_name",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "subscription_status",
+    "sourceKeys": [
+      "traits.subscriptionStatus",
+      "context.traits.subscriptionStatus"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "job_title",
+    "sourceKeys": [
+      "traits.job_title",
+      "traits.jobTitle",
+      "context.traits.job_title",
+      "context.traits.jobTitle"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "work_number",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "external_id",
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "mobile_number",
+    "sourceKeys": ["traits.mobileNumber", "context.traits.mobileNumber"],
+    "required": false
```

Comment:
Can you check if we can create a user using this or not? According to the doc we can.

Reply:
Where did you find this, can you attach some references?

Reply:
https://developers.freshworks.com/crm/api/#contact_attributes:~:text=Either%20Email/Mobile/Twitter%20ID%20is%20required%20to%20create%20a%20contact.

Reply:
Yeah, it is written in docs, but I tried to send it with mobile, external_id, and Twitter and it throws error with the message "Need to fill this Emails to submit the form."